wallet/wallet-controller/src/helpers.rs

```diff
@@ -291,42 +291,13 @@ async fn into_utxo_and_destination<T: NodeInterface, B: storage::Backend>(
             (None, TxAdditionalInfo::new(), dest)
         }
         TxInput::AccountCommand(_, cmd) => {
-            // find authority of the token
+            // find authority of the order
```

Comment:
The comment is still weird.

"find a destination associated with the order" would make more sense (not sure if it's useful though)